src/agent/communicator/src/http_socket_factory.hpp

```diff
@@ -15,7 +15,8 @@ namespace http_client
     public:
         std::unique_ptr<IHttpSocket> Create(const boost::asio::any_io_executor& executor) override
         {
-            return std::make_unique<HttpSocket>(executor);
+            //return std::make_unique<HttpSocket>(executor);
+            return std::make_unique<HttpsSocket>(executor);
```

Comment:
Looks like these changes were made to make some local tests, can they be reverted?

Reply:
Done!